### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by starlord-daniel**
Just README.md change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,5 @@ It is useful but not required to have a basic knowledge of the following topics:
 ## Resources
 
 The provisioning of this environment for proctors can be found in the [DevOps Openhack Proctor](https://github.com/Azure-Samples/openhack-devops-proctor) Github repository.
+
+Please look at the GitHub documentation!
```
Comment (Joellewoo) [inline README.md:35]: Nice change!!

Comment (starlord-daniel) [inline README.md:35]: Good!

File: README.md [hunk 2]
```diff
@@ -31,3 +31,5 @@ It is useful but not required to have a basic knowledge of the following topics:
 ## Resources
 
 The provisioning of this environment for proctors can be found in the [DevOps Openhack Proctor](https://github.com/Azure-Samples/openhack-devops-proctor) Github repository.
+
```
Comment (baikaljam) [inline README.md:34]: Will do

**Comment by starlord-daniel** [comment]
Done

**Pull request merged**